src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -340,6 +340,16 @@ public void usage(final PrintStream stream, final boolean printCommon) {
         }
     }
 
+    private String convertFromHtml(final String textToConvert) {
```

Comment:
I'll see what I can do. When you mean "fail if this happens" you mean fail the build? That would have to happen somewhere else, I think.